.github/CODEOWNERS

```diff
@@ -204,6 +204,7 @@ CHANGELOG*
 /x-pack/metricbeat/docs/ # Listed without an owner to avoid maintaining doc ownership for each input and module.
 /x-pack/metricbeat/module/activemq @elastic/obs-infraobs-integrations
 /x-pack/metricbeat/module/airflow @elastic/obs-infraobs-integrations
+/x-pack/metricbeat/module/autoops_es @elastic/opex-back
```

Comment:
@cmacknz I'm not sure what the process is for getting this approved (and thus avoiding the error above). The team is visible, so I'm guessing it's the write access.

> This CODEOWNERS file contains errors …
CODEOWNERS errors
>
> Unknown owner on line 207: make sure the team @elastic/opex-back exists, is publicly visible, and has write access to the repository
>
> …/metricbeat/module/autoops_es @elastic/opex-back

Reply:
I've added elastic/opex-back as a team with write access to Beats and that has resolved it.